src/libraries/System.Diagnostics.EventLog/src/PACKAGE.md

```diff
@@ -2,43 +2,120 @@
 
 <!-- A description of the package and where one can find more documentation -->
 
+When an error occurs in a Windows machine, the system administrator or support representative must determine what caused the error, attempt to recover any lost data, and prevent the error from recurring. It is helpful if applications, the operating system, and other system services record important events, such as low-memory conditions or excessive attempts to access a disk. The system administrator can then use the Windows Event Log to help determine what conditions caused the error and identify the context in which it occurred.
 
+This package provides the `System.Diagnostics.EventLog.dll` assembly, which contains types that allow applications to interact with the Windows Event Log service.
 
 ## Key Features
 
 <!-- The key features of this package -->
 
-*
-*
-*
+* Allows reading from existing logs.
+* Allows writing entries to logs.
+* Can create or delete event sources.
+* Can delete logs.
+* Can respond to log entries.
+* Can create new logs when creating an event source.
 
 ## How to Use
 
 <!-- A compelling example on how to use this package with code, as well as any specific guidelines for when to use the package -->
 
+```cs
+if(!EventLog.SourceExists("MySource"))
+{
+    // An event log source should not be created and immediately used.
+    // There is a latency time to enable the source, it should be created
+    // prior to executing the application that uses the source.
+    // Execute this sample a second time to use the new source.
+    EventLog.CreateEventSource("MySource", "MyNewLog");
+    Console.WriteLine("Event source created. Exiting, execute the application a second time to use the source.");
+    // The source is created. Exit the application to allow it to be registered.
+    return;
+}
+
+EventLog myLog = new();
+myLog.Source = "MySource";
+myLog.WriteEntry("Writing an informational entry to the event log.");
+```
+
+Notes:
+
+- This assembly is only supported on Windows operating systems.
+- Starting with Windows Vista, you must run the application as an administrator to interact with the Windows Event Log service using the `System.Diagnostics.EventLog` class.
+
 ## Main Types
 
 <!-- The main types provided in this library -->
 
 The main types provided by this library are:
 
-* ``
-* ``
-* ``
+Under the [`System.Diagnostics`](https://learn.microsoft.com/en-us/dotnet/api/System.Diagnostics) namespace:
+
+- [`System.Diagnostics.EntryWrittenEventArgs`](https://learn.microsoft.com/en-us/dotnet/api/System.Diagnostics.EntryWrittenEventArgs)
```

Comment:
This lists too many types, the intent of "Main types" is to include "entry point" types, i.e. ones users interact with at first when they start using library. For EventLog, i think it should list EventLog, EventLogEntry and maybe a couple other classes, but not exceptions or event handlers, they are not very interesting.

Reply:
Agreed. Another reason to not list all these types is to avoid duplication between the in-tree source code, docs.microsoft.com and this file.

Reply:
I reduced the list of types from both of the assemblies I worked on.